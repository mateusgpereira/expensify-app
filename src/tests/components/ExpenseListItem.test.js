import React from 'react'
import { shallow } from 'enzyme'
import expenses from '../fixtures/expenses'
import ExpenseListItem from '../../components/ExpenseListItem'

test('should render an ExpenseListItem', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[2]} />)
    expect(wrapper).toMatchSnapshot()
})