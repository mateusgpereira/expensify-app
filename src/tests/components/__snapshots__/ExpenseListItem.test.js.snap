// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an ExpenseListItem 1`] = `
<Link
  className="list-item"
  to="/edit/3"
>
  <div>
    <h3
      className="list-item__title"
    >
      Coffee
    </h3>
    <span
      className="list-item__subtitle"
    >
      January 4th, 1970
    </span>
  </div>
  <h3
    className="list-item__data"
  >
    $5.90
  </h3>
</Link>
`;
